scripts/releasing.js

```diff
@@ -0,0 +1,37 @@
+const { cat, exec } = require('shelljs');
```

Comment:
I wasn't aware that we have dependency for this package present in our repo. I've inspected `yarn.lock` & it seems that it is transitive dependency of our package through `react-native-windows` & `release-it`. To be honest, these are pretty stable dependencies, but I don't think we should rely on their presence. We should specify the dependency on things we're using directly.

Reply:
![image](https://github.com/user-attachments/assets/cb76344a-27ed-4551-86cb-56f549b76368)